.circleci/run_locally.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
```

Comment:
it might be useful to run `pipenv clean` and `pipenv sync` before running all of this. Just to make sure that whoever is running this has actually installed the relevant programs.